shape_completion_training/src/shape_completion_training/model/filepath_tools.py

```diff
@@ -17,79 +18,65 @@ def unique_trial_name(*names):
     return format_string.format(stamp, sha, *names)
 
 
-def get_trial_directory(base_directory, nick=None, expect_reuse=False, write_summary=True):
+def get_trial_directory(nick=None, write_summary=True):
     """
-    Returns the filepath to the directory for the trial, prompting the user for information if necessary.
-    The directory is created if it does not exist.
-
-    Call as: get_trial_directory("/path/to/trials/directory/")
+    Returns the filepath to the directory for the trial
     """
+    r = rospkg.RosPack()
+    shape_completion_training_path = r.get_path('shape_completion_training')
+    base_directory = join(shape_completion_training_path, 'trials')
     if not os.path.isdir(base_directory):
         _make_new_trials_directory(base_directory)
 
     if nick is None:
         nick = _make_tmp_nick(base_directory)
 
     unique_trial_subdirectory_name = unique_trial_name()
-    fp = join(base_directory, nick, unique_trial_subdirectory_name)
-    os.mkdir(fp)
+    fp = join(base_directory, nick)
+    if not os.path.exists(fp):
+        os.mkdir(fp)
+    full_trial_directory = join(base_directory, nick, unique_trial_subdirectory_name)
+    if os.path.exists(full_trial_directory):
+        raise RuntimeError("full trial directory already exists!?")
+    os.mkdir(full_trial_directory)
 
     if write_summary:
-        _write_summary(fp, nick)
+        _write_summary(full_trial_directory, nick)
 
-    print("Running trial {} at {}".format(nick, fp))
-    return fp
+    print("Running trial {} at {}".format(nick, full_trial_directory))
+    return full_trial_directory
 
 
-def handle_params(default_params_fp, model_params_fp, given_params):
+def handle_params(trial_directory, given_params):
```

Comment:
This needs some more work.
the docstring no longer holds, and this will not work with the existing architecture.

Somehow this (or a similar function) needs to 
```
if creating new model_model:
    choose which params to use (json, python dict, ???)
    write a json file to a model directory
return load(existing params)

```